bosh/releases/pre_render_scripts/diego-api/bbs/jobs/patch_bbs_ctl.sh

```diff
@@ -1,33 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/diego/bbs/templates/bbs_ctl.erb"
```

Comment:
… Yep.

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d